frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.js

```diff
@@ -428,6 +431,14 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
             <InfoCircleOutlined style={{ marginLeft: 4 }} />
           </Tooltip>
 
+          {intensityRange[0] === intensityRange[1] && !isDisabled ? (
+            <Tooltip
+              title={`No data is being rendered for this layer as the minimum and maximum of the range have the same values. 
+            If you do not want this layer to be rendered, better disable it with the switch on the left.`}
```

Comment:
:+1: for taking the time to handle this in the UI

```suggestion
            If you want to hide this layer, you can also disable it with the switch on the left.`}
```

sounds better imo :)